launch/app/app.bndrun

```diff
@@ -75,7 +75,9 @@ feature.openhab-model-runtime-all: \
 	osgi.console.enable.builtin=false,\
 	logback.configurationFile=file:${.}/runtime/logback.xml
 
--runblacklist: bnd.identity;id='org.apache.aries.javax.jax.rs-api'
+-runblacklist: bnd.identity;id='org.apache.aries.javax.jax.rs-api',\
+    bnd.identity;id='org.apache.aries.jpa.container',\
+    bnd.identity;id='org.openhab.core.test'
```

Comment:
Maybe also add `    bnd.identity;id='org.openhab.core.io.transport.serial.javacomm'` to the blacklist. 
It gives me the warning:
```
 [org.openhab.core.io.transport.serial.javacomm version=2.5.0-SNAPSHOT]
      ⇒ osgi.wiring.package: (&(osgi.wiring.package=javax.comm))
```

Reply:
I don't think the correct approach is to blacklist OH bundles that dependencies are currently not part of the index.

If stuff is missing, don't at it to the requirements or add the dependencies to the index.

Reply:
Well, if I do a "resolve", one of the messages is that the test bundle cannot resolve junit.
You are right, the better approach would be to make sure that the test bundle does not at all occur as a run requirement - but as I have no clue why it does, I decided to simply put it to the blacklist.
Same for the jpa container - I don't know, why it appears, but I do know that we neither want nor need it, hence I added it to the blacklist.